templates/tftmpl/integration_test.go

```diff
@@ -229,21 +229,26 @@ func TestRenderTFVarsTmpl(t *testing.T) {
 			defer cancel()
 
 			gld := testutils.CheckFile(t, true, tc.goldenFile, "")
-			retry := 0
+			var cachedContents string
 			for {
 				re, err := r.Run(tmpl, w)
 				require.NoError(t, err)
 
 				if re.Complete {
 					// there may be a race with the consul services registering
 					// let's retry once.
```

Comment:
Looks like the 'retry once' comment no longer relevant. Remove?

Reply:
Thanks for clarifying this in the comment below! 👍 the retry is still happening once.